lms/djangoapps/edxnotes/decorators.py

```diff
@@ -38,7 +43,8 @@ def get_html(self, *args, **kwargs):
         # - Harvard Annotation Tool is enabled for the course
         # - the feature flag or `edxnotes` setting of the course is set to False
         # - the user is not authenticated
-        user = self.runtime.get_real_user(self.runtime.anonymous_student_id)
+
+        user = get_current_user()
```

Comment:
Why was this change made? Is this really the same thing?

Reply:
I changed it to remove another query for the current user. As far as I can tell, the original django user is not stored in the runtime, so there's this indirection of converting the anonymous student id back into a django user. 

It's possible that this won't work the same way when masquerading as a student. I am not sure.

Reply:
If it's a relatively small part of the gain from this PR, please revert this bit. It's probably not worth the trouble of tracking down.

Reply:
Ok, I reverted it.